ui/component/or-asset-viewer/src/index.ts

```diff
@@ -221,6 +223,22 @@ export class OrAssetViewerSaveEvent extends CustomEvent<SaveResult> {
     }
 }
 
+export class OrAssetViewerChangeParentEvent extends CustomEvent<any> {
```

Comment:
Detail of the custom event shouldn't be any but `{parentId: string, assetIds: string[]}`

Reply:
right